### ISSUE TITLE
Privacy Teaching series: Introduction to Remote data science explain the tools that enable in layman terms

### ISSUE THREAD

**Issue by AbinavRavi** [OpenMined/writing#191]
Explain Lesson one tools section in course 3 in layman terms.

**Comment by eswarijayakumar** [comment]
@AbinavRavi Please assign this to me. Thanks

**Comment by Metrix1010** [comment]
Hi @eswarijayakumar can you provide a status update on this issue?

**Comment by Metrix1010** [comment]
Hi @eswarijayakumar I see you are still assigned to this issue. Did you get it published?
